s/strimzi-kafka-oauth/redhat_ubi8/build.sh

```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : strimzi-kafka-oauth
+# Version       : master (default)
+# Source repo   : https://github.com/strimzi/strimzi-kafka-oauth
+# Tested on     : RHEL_8
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Amir Sanjar <amir.sanjar@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+if [ -z $1 ] 
+then
+	# Master branch
+	BRANCH=""
```

Comment:
If no branch/tag is explicitly specified, don't we plan to clone the last stable release instead of master (which isn't always guaranteed to be stable)?

Reply:
We need a master branch for nightly CI/CD.